### PR TITLE
Bump subxt from 0.25 to 0.28

### PR DESCRIPTION
**Pull request by smiasojed**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -118,31 +121,57 @@ pub async fn subscribe_stats(
                     weight: total_weight,
                     num_extrinsics: stats.num_extrinsics,
                     max_pov: POV_MAX,
-                    max_weight: max_block_weights.max_block,
+                    max_weight: max_block_weights.max_block.ref_time,
```
Comment (ascjones) [inline src/lib.rs]: I think we should make `max_weight` and `weight` contain the full `Weight` type, not just the `ref_time`. This is useful for what we are measuring.

Comment (smiasojed) [reply src/lib.rs]: I have added full Weight type

File: src/lib.rs [hunk 2]
```diff
@@ -125,24 +131,60 @@ pub async fn subscribe_stats(
...
+}
+
+impl Add for Weight {
+	type Output = Self;
+	fn add(self, rhs: Self) -> Self {
+		Self {
+			ref_time: self.ref_time + rhs.ref_time,
+			proof_size: self.proof_size + rhs.proof_size,
+		}
+	}
```
Comment (ascjones) [inline src/lib.rs]: Please fix indentation here

**Review by ascjones — APPROVED** [review]
LGTM

**Pull request merged**